### ISSUE TITLE
crash on High Sierra 10.13.6

### ISSUE THREAD

**Issue by eleaner** [gridsync/gridsync#109]
Hi,

The app crashes immediately when launched.
Any ideas?

**Comment by crwood** [comment]
Hey there,

Thanks a bunch for reaching out -- and I'm sorry to hear that you're having troubles. For better or worse, I am unable to reproduce any launch-time crashes on my end (with either 10.13.6 or 10.9.5, testing both stable and development builds) but I might be able to help out... A few questions to try and narrow things down:

1) What version of Gridsync are you running? Did you download a binary (".dmg") package (i.e., from [GitHub Releases](https://github.com/gridsync/gridsync/releases)), or compile/install from source?

2) Can you describe what you mean by "the app crashes immediately"? Does any sort of error message show up, for example, or does the application icon simply bounce in the Dock and disappear?

3) Could the application launch have been blocked by [Gatekeeper](https://support.apple.com/en-us/HT202491)? (If you saw a popup message indicating that "_'Gridsync' can't be opened because it is from an unidentified developer_", you will have to whitelist the application by going to "_System Preferences_" -> "_Security & Privacy_" -> "_General_" and clicking the "_Open Anyway_" button). Note that Gridsync binaries are signed with PGP/GPG (and can/should be verified by following the instructions [here](https://github.com/gridsync/gridsync/releases)) but I have not yet purchased an Apple Developer signing key (and am waiting until the next release to do so).

4) Failing the above, could you try launching the application from a command line with the `--debug` flag and seeing if it displays any errors or other notable output? You can do this by opening/running Terminal.app and entering the following command: `Gridsync.app/Contents/MacOS/Gridsync --debug` (though, depending on what directory the application is in, you may need to prepend the full path -- e.g., `/Users/name/path/to/Gridsync.app/Contents/MacOS/Gridsync --debug`)

**Comment by eleaner** [comment]
Thank you for responding
1. I am running v0.3.2 from an image downloaded of github
2. it means I immediately get pop up "Gridsync quit unexpectedly." with options to ignore/report or reopen
3. I guess it could but I would think that OSX would handle such a situation gracefully. 
4. the output is simple. Just one line "Illegal instruction: 4"

Thanks,
Marcin

**Comment by crwood** [comment]
Thank you so much for the responses!

The only time I've seen the "Illegal instruction: 4" error was on very old hardware -- at the time, a 2007 iMac, which, apparently, lacked some modern CPU instruction set used by one of the dependencies (I believe, PyNaCl/libsodium). That said, what year/model is your mac? And can you paste the output of the crash log (i.e.,by clicking the "Report" button and then pasting the output here)?

**Comment by eleaner** [comment]
That's interesting

I am running early 2009 MacPro flashed to 5.1
if the old hardware is the reason gridsync would be the first software I
encountered that relies on hardware more than on OS.
I am using Mac precisely to avoid this issue (as long as OS is supported)

Process:               Gridsync [74580]
Path:                  /Applications/Gridsync.app/Contents/MacOS/Gridsync
Identifier:            io.gridsync.Gridsync
Version:               0.0.0 (???)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           Gridsync [74580]
User ID:               501

Date/Time:             2018-08-12 20:39:58.761 +0100
OS Version:            Mac OS X 10.13.6 (17G65)
Report Version:        12
Anonymous UUID:        5C496799-8B65-5F3F-FFBF-D31318554445

Sleep/Wake UUID:       9C5D87F7-A26A-4B16-BF6B-7D6A9E4877E7

Time Awake Since Boot: 39000 seconds
Time Since Wake:       22000 seconds

System Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_INSTRUCTION (SIGILL)
Exception Codes:       0x0000000000000001, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Signal:    Illegal instruction: 4
Termination Reason:    Namespace SIGNAL, Code 0x4
Terminating Process:   exc handler [0]

Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   .Python                       0x0000000105108659 ucs4lib_utf8_decode +
51
1   .Python                       0x0000000105108b1a
_Py_DecodeUTF8_surrogateescape + 110
2   .Python                       0x0000000105183780 decode_locale + 244
3   io.gridsync.Gridsync           0x0000000102cf26dc 0x102cef000 + 14044
4   io.gridsync.Gridsync           0x0000000102cf2786 0x102cef000 + 14214
5   io.gridsync.Gridsync           0x0000000102cf12f6 0x102cef000 + 8950
6   io.gridsync.Gridsync           0x0000000102cf169a 0x102cef000 + 9882
7   io.gridsync.Gridsync           0x0000000102cefec4 0x102cef000 + 3780

Thread 1:
0   libsystem_kernel.dylib         0x00007fff7213a28a __workq_kernreturn +
10
1   libsystem_pthread.dylib       0x00007fff72301009 _pthread_wqthread +
1035
2   libsystem_pthread.dylib       0x00007fff72300be9 start_wqthread + 13

Thread 2:
0   libsystem_kernel.dylib         0x00007fff7213a28a __workq_kernreturn +
10
1   libsystem_pthread.dylib       0x00007fff7230120e _pthread_wqthread +
1552
2   libsystem_pthread.dylib       0x00007fff72300be9 start_wqthread + 13

Thread 3:
0   libsystem_kernel.dylib         0x00007fff7213a28a __workq_kernreturn +
10
1   libsystem_pthread.dylib       0x00007fff72301009 _pthread_wqthread +
1035
2   libsystem_pthread.dylib       0x00007fff72300be9 start_wqthread + 13

Thread 4:
0   libsystem_kernel.dylib         0x00007fff7213a28a __workq_kernreturn +
10
1   libsystem_pthread.dylib       0x00007fff72301009 _pthread_wqthread +
1035
2   libsystem_pthread.dylib       0x00007fff72300be9 start_wqthread + 13

Thread 0 crashed with X86 Thread State (64-bit):
  rax: 0x0000000000000000  rbx: 0x00007f90ef802eaa  rcx: 0x00007ffeecf0ed80
 rdx: 0x000062c00018b870
  rdi: 0x00007ffeecf0ed88  rsi: 0x00007f90ef802eaa  rbp: 0x00007ffeecf0ed70
 rsp: 0x00007ffeecf0ed48
   r8: 0x000062c00018b870   r9: 0x00007f90ef802e78  r10: 0x00007f90ef802ea8
 r11: 0x8080808080808080
  r12: 0x00007ffeecf0ed80  r13: 0x00007ffeecf10390  r14: 0x000062c00018b870
 r15: 0x00007ffeecf0ed88
  rip: 0x0000000105108659  rfl: 0x0000000000010202  cr2: 0x00000001050e9f7c

Logical CPU:     22
Error Code:      0x00000000
Trap Number:     6


Binary Images:
       0x102cef000 -        0x102cf5ff7 +io.gridsync.Gridsync (0.0.0 - ???)
<32AB4B3F-56DF-3C81-B3EE-2215AC410955>
/Applications/Gridsync.app/Contents/MacOS/Gridsync
       0x1050a0000 -        0x105218fef +.Python (0)
<4E11B463-32F5-378D-BCB3-C9772F6949A0>
/Applications/Gridsync.app/Contents/MacOS/.Python
       0x109eb0000 -        0x109efaacf  dyld (551.4)
<8A72DE9C-A136-3506-AA02-4BA2B82DCAF3> /usr/lib/dyld
    0x7fff46371000 -     0x7fff46371fff  com.apple.Accelerate (1.11 -
Accelerate 1.11) <2EF00EFE-06E8-3082-85C3-2988B422A19D>
/System/Library/Frameworks/Accelerate.framework/Versions/A/Accelerate
    0x7fff4638a000 -     0x7fff46ad8ffb  com.apple.vImage (8.1 - ???)
<56C275C1-459F-37CD-BF29-2E6D81C29E53>
/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vImage.framework/Versions/A/vImage
    0x7fff46ad9000 -     0x7fff46c79ff3  libBLAS.dylib (1211.50.2)
<93259222-2D94-333D-A1EB-09C6FC4A64B4>
/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
    0x7fff46c7a000 -     0x7fff46cb4fef  libBNNS.dylib (38.1)
<0AAE0109-71D5-3B26-8401-3E62DDF97624>
/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBNNS.dylib
    0x7fff46cb5000 -     0x7fff4708eff7  libLAPACK.dylib (1211.50.2)
<70781D0B-AEAD-33EB-9900-10DC30F37D61>
/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libLAPACK.dylib
    0x7fff4708f000 -     0x7fff470a5ff7  libLinearAlgebra.dylib (1211.50.2)
<FB5A35EA-D636-3D69-AB53-B67E1EB8134C>
/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libLinearAlgebra.dylib
    0x7fff470a6000 -     0x7fff470abff3  libQuadrature.dylib (3)
<3D6BF66A-55B2-3692-BAC7-DEB0C676ED29>
/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libQuadrature.dylib
    0x7fff470ac000 -     0x7fff4712cfff  libSparse.dylib (79.50.2)
<0DC25CDD-F8C1-3D6E-B472-8B060708424F>
/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libSparse.dylib
    0x7fff4712d000 -     0x7fff4713fff7  libSparseBLAS.dylib (1211.50.2)
<994A7D7E-FA3B-3943-A59C-8150B9AF86BE>
/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libSparseBLAS.dylib
    0x7fff47140000 -     0x7fff472b3fe7  libvDSP.dylib (622.50.5)
<4BF310F4-31A3-3DA5-80E4-7F8014AD380B>
/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libvDSP.dylib
    0x7fff472b4000 -     0x7fff4736afef  libvMisc.dylib (622.50.5)
<88143A88-4FDE-35A1-85FE-54FF0C2A9E43>
/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libvMisc.dylib
    0x7fff4736b000 -     0x7fff4736bfff  com.apple.Accelerate.vecLib (3.11
- vecLib 3.11) <880DEF40-389A-3F56-ACA1-B62A5BC57E59>
/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/vecLib
    0x7fff4850f000 -     0x7fff4850ffff  com.apple.ApplicationServices (48
- 50) <AFFBD94A-AF76-336E-B53E-57524EAE8EF3>
/System/Library/Frameworks/ApplicationServices.framework/Versions/A/ApplicationServices
    0x7fff48510000 -     0x7fff48576fff  com.apple.ApplicationServices.ATS
(377 - 445.4) <85E779EE-0219-3181-B4C4-201E4CC82AB5>
/System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/ATS.framework/Versions/A/ATS
    0x7fff4860f000 -     0x7fff48731fff  libFontParser.dylib (222.1.6)
<6CEBACDD-B848-302E-B4B2-630CB16E663E>
/System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/ATS.framework/Versions/A/Resources/libFontParser.dylib
    0x7fff48732000 -     0x7fff4877cff7  libFontRegistry.dylib (221.4)
<5FDB4F1A-E15C-3ACB-A5C1-F15458C0C6DC>
/System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/ATS.framework/Versions/A/Resources/libFontRegistry.dylib
    0x7fff488be000 -     0x7fff488c2ff3  com.apple.ColorSyncLegacy (4.13.0
- 1) <A5FB2694-1559-34A8-A3D3-2029F68A63CA>
/System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/ColorSyncLegacy.framework/Versions/A/ColorSyncLegacy
    0x7fff48962000 -     0x7fff489b4ffb  com.apple.HIServices (1.22 -
624.1) <66FD9ED2-9630-313C-86AE-4C2FBCB3F351>
/System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/HIServices.framework/Versions/A/HIServices
    0x7fff489b5000 -     0x7fff489c3fff  com.apple.LangAnalysis (1.7.0 -
1.7.0) <B65FF7E6-E9B5-34D8-8CA7-63D415A8A9A6>
/System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/LangAnalysis.framework/Versions/A/LangAnalysis
    0x7fff489c4000 -     0x7fff48a10fff
 com.apple.print.framework.PrintCore (13.4 - 503.2)
<B90C67C1-0292-3CEC-885D-F1882CD104BE>
/System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/PrintCore.framework/Versions/A/PrintCore
    0x7fff48a11000 -     0x7fff48a4bfff  com.apple.QD (3.12 - 404.2)
<38B20AFF-9D54-3B52-A6DC-C0D71380AA5F>
/System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/QD.framework/Versions/A/QD
    0x7fff48a4c000 -     0x7fff48a58fff
 com.apple.speech.synthesis.framework (7.8.1 - 7.8.1)
<A08DE016-C8F2-3B0E-BD34-15959D13DBF0>
/System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/SpeechSynthesis.framework/Versions/A/SpeechSynthesis
    0x7fff48a59000 -     0x7fff48ce7ff7
 com.apple.audio.toolbox.AudioToolbox (1.14 - 1.14)
<E0B8B5D8-80A0-308B-ABD6-F8612102B5D8>
/System/Library/Frameworks/AudioToolbox.framework/Versions/A/AudioToolbox
    0x7fff48ce9000 -     0x7fff48ce9fff  com.apple.audio.units.AudioUnit
(1.14 - 1.14) <ABF8778E-4F9D-305E-A528-DE406A1A2B68>
/System/Library/Frameworks/AudioUnit.framework/Versions/A/AudioUnit
    0x7fff4900c000 -     0x7fff493a6ff7  com.apple.CFNetwork (902.1 -
902.1) <76EB8CB6-BF59-3BDA-BF2B-F21B161611B9>
/System/Library/Frameworks/CFNetwork.framework/Versions/A/CFNetwork
    0x7fff493bb000 -     0x7fff493bbfff  com.apple.Carbon (158 - 158)
<F8B370D9-2103-3276-821D-ACC756167F86>
/System/Library/Frameworks/Carbon.framework/Versions/A/Carbon
    0x7fff493bc000 -     0x7fff493bfffb  com.apple.CommonPanels (1.2.6 -
98) <2391761C-5CAA-3F68-86B7-50B37927B104>
/System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/CommonPanels.framework/Versions/A/CommonPanels
    0x7fff493c0000 -     0x7fff496c5fff  com.apple.HIToolbox (2.1.1 -
911.10) <BF7F9C0E-C732-3FB2-9BBC-362888BDA57B>
/System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/HIToolbox.framework/Versions/A/HIToolbox
    0x7fff496c6000 -     0x7fff496c9ffb  com.apple.help (1.3.8 - 66)
<DEBADFA8-C189-3195-B0D6-A1F2DE95882A>
/System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/Help.framework/Versions/A/Help
    0x7fff496ca000 -     0x7fff496cffff  com.apple.ImageCapture (9.0 - 9.0)
<23B4916F-3B43-3DFF-B956-FC390EECA284>
/System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/ImageCapture.framework/Versions/A/ImageCapture
    0x7fff496d0000 -     0x7fff49765ffb  com.apple.ink.framework (10.9 -
221) <5206C8B0-22DA-36C9-998E-846EDB626D5B>
/System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/Ink.framework/Versions/A/Ink
    0x7fff49766000 -     0x7fff49780ff7  com.apple.openscripting (1.7 -
174) <1B2A1F9E-5534-3D61-83CA-9199B39E8708>
/System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/OpenScripting.framework/Versions/A/OpenScripting
    0x7fff497a1000 -     0x7fff497a2fff  com.apple.print.framework.Print
(12 - 267) <3682ABFB-2561-3419-847D-02C247F4800D>
/System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/Print.framework/Versions/A/Print
    0x7fff497a3000 -     0x7fff497a5ff7  com.apple.securityhi (9.0 - 55006)
<C1406B8D-7D05-3959-808F-9C82189CF57F>
/System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/SecurityHI.framework/Versions/A/SecurityHI
    0x7fff497a6000 -     0x7fff497acfff
 com.apple.speech.recognition.framework (6.0.3 - 6.0.3)
<2ED8643D-B0C3-3F17-82A2-BBF13E6CBABC>
/System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/SpeechRecognition.framework/Versions/A/SpeechRecognition
    0x7fff498db000 -     0x7fff49994fff  com.apple.ColorSync (4.13.0 -
3325) <D283C285-447D-3258-A7E4-59532123B8FF>
/System/Library/Frameworks/ColorSync.framework/Versions/A/ColorSync
    0x7fff49b21000 -     0x7fff49bb4ff7  com.apple.audio.CoreAudio (4.3.0 -
4.3.0) <EB35D3EC-56EA-33E6-98DC-BDC3A5FA8ACE>
/System/Library/Frameworks/CoreAudio.framework/Versions/A/CoreAudio
    0x7fff49c1b000 -     0x7fff49c44ffb  com.apple.CoreBluetooth (1.0 - 1)
<E1335074-9D07-370E-8440-61C4874BAC56>
/System/Library/Frameworks/CoreBluetooth.framework/Versions/A/CoreBluetooth
    0x7fff49c45000 -     0x7fff49fa9ff3  com.apple.CoreData (120 - 851)
<8D605466-8676-3D45-9A4B-8980971551C3>
/System/Library/Frameworks/CoreData.framework/Versions/A/CoreData
    0x7fff49faa000 -     0x7fff4a08dfff  com.apple.CoreDisplay (99.14 -
99.14) <A1B91ADD-828D-33A0-8A92-CC3F83DF89D0>
/System/Library/Frameworks/CoreDisplay.framework/Versions/A/CoreDisplay
    0x7fff4a08e000 -     0x7fff4a528ff7  com.apple.CoreFoundation (6.9 -
1454.90) <F320589A-8600-3244-9B55-A645F7EAAB35>
/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
    0x7fff4a52a000 -     0x7fff4ab58ff7  com.apple.CoreGraphics (2.0 -
1161.21) <C4CBD2CF-39D5-3185-851E-85C7DD2F8C7F>
/System/Library/Frameworks/CoreGraphics.framework/Versions/A/CoreGraphics
    0x7fff4ab5a000 -     0x7fff4ae49fff  com.apple.CoreImage (13.0.0 -
579.5) <AAE2DFD0-9B0A-3D56-8A3E-C460BAF70394>
/System/Library/Frameworks/CoreImage.framework/Versions/A/CoreImage
    0x7fff4b21e000 -     0x7fff4b21efff  com.apple.CoreServices (822.36 -
822.36) <C8368F17-1589-3BA5-A0E7-89CB8DF2454F>
/System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices
    0x7fff4b21f000 -     0x7fff4b293ffb  com.apple.AE (735.1 - 735.1)
<08EBA184-20F7-3725-AEA6-C314448161C6>
/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/AE.framework/Versions/A/AE
    0x7fff4b294000 -     0x7fff4b56bfff  com.apple.CoreServices.CarbonCore
(1178.4 - 1178.4) <0D5E19BF-18CB-3FA4-8A5F-F6C787C5EE08>
/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/CarbonCore.framework/Versions/A/CarbonCore
    0x7fff4b56c000 -     0x7fff4b5a0fff  com.apple.DictionaryServices (1.2
- 284.2) <6505B075-41C3-3C62-A4C3-85CE3F6825CD>
/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/DictionaryServices.framework/Versions/A/DictionaryServices
    0x7fff4b5a1000 -     0x7fff4b5a9ffb  com.apple.CoreServices.FSEvents
(1239.50.1 - 1239.50.1) <3637CEC7-DF0E-320E-9634-44A442925C65>
/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/FSEvents.framework/Versions/A/FSEvents
    0x7fff4b5aa000 -     0x7fff4b767fff  com.apple.LaunchServices (822.36 -
822.36) <6E68C090-B12D-3D3D-9617-E5D82C36B2D0>
/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/LaunchServices.framework/Versions/A/LaunchServices
    0x7fff4b768000 -     0x7fff4b818ff7  com.apple.Metadata (10.7.0 -
1191.4.13) <B5C22E70-C265-3C9F-865F-B138994A418D>
/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/Metadata.framework/Versions/A/Metadata
    0x7fff4b819000 -     0x7fff4b879fff  com.apple.CoreServices.OSServices
(822.36 - 822.36) <3BB2E0CE-81AE-3D3D-9FCE-E1B7FC6D6A61>
/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/OSServices.framework/Versions/A/OSServices
    0x7fff4b87a000 -     0x7fff4b8e8fff  com.apple.SearchKit (1.4.0 -
1.4.0) <3662545A-B1CF-3079-BDCD-C83855CEFEEE>
/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/SearchKit.framework/Versions/A/SearchKit
    0x7fff4b8e9000 -     0x7fff4b90dffb
 com.apple.coreservices.SharedFileList (71.21 - 71.21)
<35582D88-5975-35E2-A29A-E3148C3EE727>
/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/SharedFileList.framework/Versions/A/SharedFileList
    0x7fff4bbae000 -     0x7fff4bcfefff  com.apple.CoreText (352.0 -
578.22) <6129F39D-284D-3BBF-8999-7854AB61C01C>
/System/Library/Frameworks/CoreText.framework/Versions/A/CoreText
    0x7fff4bcff000 -     0x7fff4bd39fff  com.apple.CoreVideo (1.8 - 0.0)
<86CCC036-51BB-3DD1-9601-D93798BCCD0F>
/System/Library/Frameworks/CoreVideo.framework/Versions/A/CoreVideo
    0x7fff4bd3a000 -     0x7fff4bdc5ff3  com.apple.framework.CoreWLAN (13.0
- 1350.1) <E862CC02-69D2-3503-887B-B6E8223081E7>
/System/Library/Frameworks/CoreWLAN.framework/Versions/A/CoreWLAN
    0x7fff4c040000 -     0x7fff4c045fff  com.apple.DiskArbitration (2.7 -
2.7) <A975AD56-4CD3-3A89-8732-858CA9BD3DAA>
/System/Library/Frameworks/DiskArbitration.framework/Versions/A/DiskArbitration
    0x7fff4c206000 -     0x7fff4c5ccfff  com.apple.Foundation (6.9 -
1454.90) <8EA924F3-ADAE-3F4B-8482-8B11C027D9A5>
/System/Library/Frameworks/Foundation.framework/Versions/C/Foundation
    0x7fff4c63d000 -     0x7fff4c66dfff  com.apple.GSS (4.0 - 2.0)
<D774A165-5581-3479-AB5D-2BBDB5CF8882>
/System/Library/Frameworks/GSS.framework/Versions/A/GSS
    0x7fff4c77f000 -     0x7fff4c883ffb  com.apple.Bluetooth (6.0.7 -
6.0.7f10) <557F26F9-C7A0-34EA-A905-22E243BF6B48>
/System/Library/Frameworks/IOBluetooth.framework/Versions/A/IOBluetooth
    0x7fff4c8e3000 -     0x7fff4c97efff  com.apple.framework.IOKit (2.0.2 -
1445.71.1) <2EA4F383-CAA9-3AF0-99C5-90C22ADAA6B6>
/System/Library/Frameworks/IOKit.framework/Versions/A/IOKit
    0x7fff4c980000 -     0x7fff4c987fff  com.apple.IOSurface (211.15 -
211.15) <9FD406F1-6BF2-35B0-8339-DF83A1A661EB>
/System/Library/Frameworks/IOSurface.framework/Versions/A/IOSurface
    0x7fff4c9de000 -     0x7fff4cb5bfff  com.apple.ImageIO.framework (3.3.0
- 1739.3) <86D14377-8F57-387E-982E-2EF60EE41E2F>
/System/Library/Frameworks/ImageIO.framework/Versions/A/ImageIO
    0x7fff4cb5c000 -     0x7fff4cb60ffb  libGIF.dylib (1739.3)
<E1B16861-7818-3319-9EE5-7AF2BC02E4AB>
/System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libGIF.dylib
    0x7fff4cb61000 -     0x7fff4cc48fff  libJP2.dylib (1739.3)
<3C1150F8-4983-3380-9B8C-5021AD972FFE>
/System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libJP2.dylib
    0x7fff4cc49000 -     0x7fff4cc6cfff  libJPEG.dylib (1739.3)
<D20353C9-8B51-386E-97A3-B1FE0038E9B6>
/System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libJPEG.dylib
    0x7fff4cf49000 -     0x7fff4cf6fff3  libPng.dylib (1739.3)
<97A8B816-7A79-3CB8-9D47-CCDC19B84E2F>
/System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libPng.dylib
    0x7fff4cf70000 -     0x7fff4cf72ff3  libRadiance.dylib (1739.3)
<47A0D85B-8AE8-3FF2-9C13-279550850A24>
/System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libRadiance.dylib
    0x7fff4cf73000 -     0x7fff4cfc0ff3  libTIFF.dylib (1739.3)
<9833CCBB-48FB-3E7B-A11A-F2F331FD094D>
/System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libTIFF.dylib
    0x7fff4de7a000 -     0x7fff4de93ff7  com.apple.Kerberos (3.0 - 1)
<F86DCCDF-93C1-38B3-82C2-477C12E8EE6D>
/System/Library/Frameworks/Kerberos.framework/Versions/A/Kerberos
    0x7fff4e875000 -     0x7fff4e8f6fff  com.apple.Metal (125.30 - 125.30)
<975FD6B5-D695-346A-869F-0584A968D100>
/System/Library/Frameworks/Metal.framework/Versions/A/Metal
    0x7fff4e913000 -     0x7fff4e92efff
 com.apple.MetalPerformanceShaders.MPSCore (1.0 - 1)
<AD754E8F-CA00-3878-9AF3-208C224A230B>
/System/Library/Frameworks/MetalPerformanceShaders.framework/Frameworks/MPSCore.framework/Versions/A/MPSCore
    0x7fff4e92f000 -     0x7fff4e99efef
 com.apple.MetalPerformanceShaders.MPSImage (1.0 - 1)
<338B7779-E608-3D68-8A07-2ACC11299744>
/System/Library/Frameworks/MetalPerformanceShaders.framework/Frameworks/MPSImage.framework/Versions/A/MPSImage
    0x7fff4e99f000 -     0x7fff4e9c3fff
 com.apple.MetalPerformanceShaders.MPSMatrix (1.0 - 1)
<9CE072D7-853B-3939-9645-7EB951376B87>
/System/Library/Frameworks/MetalPerformanceShaders.framework/Frameworks/MPSMatrix.framework/Versions/A/MPSMatrix
    0x7fff4e9c4000 -     0x7fff4eaabff7
 com.apple.MetalPerformanceShaders.MPSNeuralNetwork (1.0 - 1)
<0DE891AD-27E5-38FF-AEC8-4A95356C4357>
/System/Library/Frameworks/MetalPerformanceShaders.framework/Frameworks/MPSNeuralNetwork.framework/Versions/A/MPSNeuralNetwork
    0x7fff4eaac000 -     0x7fff4eaacff7
 com.apple.MetalPerformanceShaders.MetalPerformanceShaders (1.0 - 1)
<2D2D261C-50B0-32F9-BF9A-5C01382BB528>
/System/Library/Frameworks/MetalPerformanceShaders.framework/Versions/A/MetalPerformanceShaders
    0x7fff4faab000 -     0x7fff4fab7ffb  com.apple.NetFS (6.0 - 4.0)
<471DD96F-FA2E-3FE9-9746-2519A6780D1A>
/System/Library/Frameworks/NetFS.framework/Versions/A/NetFS
    0x7fff528a9000 -     0x7fff52903ff7  com.apple.opencl (2.8.24 - 2.8.24)
<4D7401A7-6ADD-3632-85AE-7A5012DFFA04>
/System/Library/Frameworks/OpenCL.framework/Versions/A/OpenCL
    0x7fff52904000 -     0x7fff52920ffb  com.apple.CFOpenDirectory (10.13 -
207.50.1) <29F55F7B-379F-3053-8FF3-5C6675A3DD4D>
/System/Library/Frameworks/OpenDirectory.framework/Versions/A/Frameworks/CFOpenDirectory.framework/Versions/A/CFOpenDirectory
    0x7fff52921000 -     0x7fff5292cfff  com.apple.OpenDirectory (10.13 -
207.50.1) <F895547D-4915-353F-9C1E-E95172BA803B>
/System/Library/Frameworks/OpenDirectory.framework/Versions/A/OpenDirectory
    0x7fff53aab000 -     0x7fff53aadfff  libCVMSPluginSupport.dylib
(16.7.4) <F9270AE0-CC3B-3E3E-BA32-CC1068DD8F27>
/System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libCVMSPluginSupport.dylib
    0x7fff53aae000 -     0x7fff53ab3ffb  libCoreFSCache.dylib (162.9)
<7AF87F3E-D5D0-3625-BE09-CA4223195466>
/System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libCoreFSCache.dylib
    0x7fff53ab4000 -     0x7fff53ab8fff  libCoreVMClient.dylib (162.9)
<115FE643-6141-39B4-8193-77DFCBE7A4E0>
/System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libCoreVMClient.dylib
    0x7fff53ab9000 -     0x7fff53ac2ff3  libGFXShared.dylib (16.7.4)
<EB2BF8A0-E10D-35EA-8F46-B2E3C62C12A8>
/System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGFXShared.dylib
    0x7fff53ac3000 -     0x7fff53acefff  libGL.dylib (16.7.4)
<2BB333D3-5C61-33DF-8545-06DF2D08B83D>
/System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGL.dylib
    0x7fff53acf000 -     0x7fff53b0afe7  libGLImage.dylib (16.7.4)
<4DA003CE-0B74-3FE4-808C-B2FBCE517EB4>
/System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGLImage.dylib
    0x7fff53c79000 -     0x7fff53cb7ffb  libGLU.dylib (16.7.4)
<BCB09CD8-EB0E-38FA-8B5A-9E29532EE364>
/System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGLU.dylib
    0x7fff5462f000 -     0x7fff5463eff3  com.apple.opengl (16.7.4 - 16.7.4)
<9BDE8FF9-5418-3C70-8D1C-09656884CE48>
/System/Library/Frameworks/OpenGL.framework/Versions/A/OpenGL
    0x7fff5548d000 -     0x7fff556d9ff7  com.apple.QuartzCore (1.11 -
584.62) <1950D993-DE48-3C97-95A5-66D98BDFC95D>
/System/Library/Frameworks/QuartzCore.framework/Versions/A/QuartzCore
    0x7fff55f0e000 -     0x7fff56239fff  com.apple.security (7.0 -
58286.70.7) <9FC166E1-14D0-305C-A086-02B9E83F547E>
/System/Library/Frameworks/Security.framework/Versions/A/Security
    0x7fff5623a000 -     0x7fff562c6ff7  com.apple.securityfoundation (6.0
- 55185.50.5) <D708D069-AEDB-36C2-B1DA-479DA91D7711>
/System/Library/Frameworks/SecurityFoundation.framework/Versions/A/SecurityFoundation
    0x7fff562f8000 -     0x7fff562fcffb  com.apple.xpc.ServiceManagement
(1.0 - 1) <71B45D83-ECA4-3265-997E-683A8B8DF413>
/System/Library/Frameworks/ServiceManagement.framework/Versions/A/ServiceManagement
    0x7fff566a1000 -     0x7fff56711ff3  com.apple.SystemConfiguration
(1.17 - 1.17) <8532B8E9-7E30-35A3-BC4A-DDE8E0614FDA>
/System/Library/Frameworks/SystemConfiguration.framework/Versions/A/SystemConfiguration
    0x7fff5960e000 -     0x7fff596a1fff  com.apple.APFS (1.0 - 1)
<6BBB3988-1C91-314F-A77A-4E093A1B18F0>
/System/Library/PrivateFrameworks/APFS.framework/Versions/A/APFS
    0x7fff5a2cc000 -     0x7fff5a2f4fff  com.apple.framework.Apple80211
(13.0 - 1361.7) <16627876-8CF5-3502-A1D6-35FCBDD5E79A>
/System/Library/PrivateFrameworks/Apple80211.framework/Versions/A/Apple80211
    0x7fff5a404000 -     0x7fff5a40fff7  com.apple.AppleIDAuthSupport (1.0
- 1) <2FAF5567-CDB3-33EF-AB71-05D37F2248B7>
/System/Library/PrivateFrameworks/AppleIDAuthSupport.framework/Versions/A/AppleIDAuthSupport
    0x7fff5a449000 -     0x7fff5a491ff3  com.apple.AppleJPEG (1.0 - 1)
<8DD410CB-76A1-3F22-9A9F-0491FA0CEB4A>
/System/Library/PrivateFrameworks/AppleJPEG.framework/Versions/A/AppleJPEG
    0x7fff5a4cc000 -     0x7fff5a4f4fff  com.apple.applesauce (1.0 - ???)
<CCA8B094-1BCE-3AE3-A0A7-D544C818DE36>
/System/Library/PrivateFrameworks/AppleSauce.framework/Versions/A/AppleSauce
    0x7fff5a946000 -     0x7fff5abdfffb  com.apple.AuthKit (1.0 - 1)
<6CA71A11-91C5-307C-B933-9FCDEDCB580A>
/System/Library/PrivateFrameworks/AuthKit.framework/Versions/A/AuthKit
    0x7fff5ad16000 -     0x7fff5ad1dff7
 com.apple.coreservices.BackgroundTaskManagement (1.0 - 57.1)
<51A41CA3-DB1D-3380-993E-99C54AEE518E>
/System/Library/PrivateFrameworks/BackgroundTaskManagement.framework/Versions/A/BackgroundTaskManagement
    0x7fff5c75f000 -     0x7fff5c768ff3  com.apple.CommonAuth (4.0 - 2.0)
<4D237B25-27E5-3577-948B-073659F6D3C0>
/System/Library/PrivateFrameworks/CommonAuth.framework/Versions/A/CommonAuth
    0x7fff5d26e000 -     0x7fff5d27eff7  com.apple.CoreEmoji (1.0 - 69.3)
<A4357F5C-0C38-3A61-B456-D7321EB2CEE5>
/System/Library/PrivateFrameworks/CoreEmoji.framework/Versions/A/CoreEmoji
    0x7fff5da12000 -     0x7fff5da1aff3  com.apple.CorePhoneNumbers (1.0 -
1) <A5D41251-9F38-3AB9-9DE7-F77023FAAA44>
/System/Library/PrivateFrameworks/CorePhoneNumbers.framework/Versions/A/CorePhoneNumbers
    0x7fff5dba5000 -     0x7fff5dbd6ff3  com.apple.CoreServicesInternal
(309.1 - 309.1) <4ECD14EA-A493-3B84-A32F-CF928474A405>
/System/Library/PrivateFrameworks/CoreServicesInternal.framework/Versions/A/CoreServicesInternal
    0x7fff5df13000 -     0x7fff5dfa4fff  com.apple.CoreSymbolication (9.3 -
64026.2) <D55A6E5B-0267-3F3A-8D90-4B8F39458420>
/System/Library/PrivateFrameworks/CoreSymbolication.framework/Versions/A/CoreSymbolication
    0x7fff5e027000 -     0x7fff5e15cfff  com.apple.coreui (2.1 - 494.1)
<B2C515C3-FCE8-3B28-A225-05AD917F509B>
/System/Library/PrivateFrameworks/CoreUI.framework/Versions/A/CoreUI
    0x7fff5e15d000 -     0x7fff5e28efff  com.apple.CoreUtils (5.6 - 560.11)
<1A02D6F0-8C65-3FAE-AD63-56477EDE4773>
/System/Library/PrivateFrameworks/CoreUtils.framework/Versions/A/CoreUtils
    0x7fff5e2e3000 -     0x7fff5e347fff  com.apple.framework.CoreWiFi (13.0
- 1350.1) <6EC5DEB3-6E2F-3DC2-BE59-1FD05175FB0C>
/System/Library/PrivateFrameworks/CoreWiFi.framework/Versions/A/CoreWiFi
    0x7fff5e348000 -     0x7fff5e358ff7  com.apple.CrashReporterSupport
(10.13 - 1) <A909F468-0648-3F51-A77E-3F9ADBC9A941>
/System/Library/PrivateFrameworks/CrashReporterSupport.framework/Versions/A/CrashReporterSupport
    0x7fff5e3e8000 -     0x7fff5e3ecffb  com.apple.DSExternalDisplay (3.1 -
380) <901B7F6D-376A-3848-99D0-170C4D00F776>
/System/Library/PrivateFrameworks/DSExternalDisplay.framework/Versions/A/DSExternalDisplay
    0x7fff5e532000 -     0x7fff5e572ff7  com.apple.DebugSymbols (181.0 -
181.0) <299A0238-ED78-3676-B131-274D972824AA>
/System/Library/PrivateFrameworks/DebugSymbols.framework/Versions/A/DebugSymbols
    0x7fff5e573000 -     0x7fff5e6a2fff  com.apple.desktopservices (1.12.5
- 1.12.5) <7739C9A5-64D9-31A5-899B-5FFA242AD70D>
/System/Library/PrivateFrameworks/DesktopServicesPriv.framework/Versions/A/DesktopServicesPriv
    0x7fff5f4bc000 -     0x7fff5f8eafff  com.apple.vision.FaceCore (3.3.2 -
3.3.2) <B574FE33-4A41-3611-9738-388EBAF03E37>
/System/Library/PrivateFrameworks/FaceCore.framework/Versions/A/FaceCore
    0x7fff62bb2000 -     0x7fff62bb7fff  com.apple.GPUWrangler (3.20.13 -
3.20.13) <9C5BD618-69E3-36D5-9BC9-A4841BC00D2A>
/System/Library/PrivateFrameworks/GPUWrangler.framework/Versions/A/GPUWrangler
    0x7fff6392d000 -     0x7fff6393cfff  com.apple.GraphVisualizer (1.0 -
5) <B993B8A2-5700-3DFC-9EB7-4CCEE8F959F1>
/System/Library/PrivateFrameworks/GraphVisualizer.framework/Versions/A/GraphVisualizer
    0x7fff639bf000 -     0x7fff63a33fff  com.apple.Heimdal (4.0 - 2.0)
<93091531-CC91-34FF-8B93-5D3F02C37BC5>
/System/Library/PrivateFrameworks/Heimdal.framework/Versions/A/Heimdal
    0x7fff64343000 -     0x7fff6434aff7  com.apple.IOAccelerator (378.26 -
378.26) <2274BE11-18DE-3B13-BCDB-C488C9BB19AD>
/System/Library/PrivateFrameworks/IOAccelerator.framework/Versions/A/IOAccelerator
    0x7fff6434e000 -     0x7fff64365fff  com.apple.IOPresentment (1.0 -
35.1) <7C6332FF-6535-3064-B437-1E9F70671927>
/System/Library/PrivateFrameworks/IOPresentment.framework/Versions/A/IOPresentment
    0x7fff64730000 -     0x7fff64756ffb  com.apple.IconServices (97.6 -
97.6) <A56D826D-20D2-34BE-AACC-A80CFCB4E915>
/System/Library/PrivateFrameworks/IconServices.framework/Versions/A/IconServices
    0x7fff649db000 -     0x7fff649eeff3
 com.apple.security.KeychainCircle.KeychainCircle (1.0 - 1)
<AED421B0-90A0-3969-98A4-CCBCF2D3360B>
/System/Library/PrivateFrameworks/KeychainCircle.framework/Versions/A/KeychainCircle
    0x7fff649ef000 -     0x7fff64ae4ff7  com.apple.LanguageModeling (1.0 -
159.5.3) <7F0AC200-E3DD-39FB-8A95-00DD70B66A9F>
/System/Library/PrivateFrameworks/LanguageModeling.framework/Versions/A/LanguageModeling
    0x7fff64ae5000 -     0x7fff64b27fff  com.apple.Lexicon-framework (1.0 -
33.5) <DC94CF9E-1EB4-3C0E-B298-CA1190885276>
/System/Library/PrivateFrameworks/Lexicon.framework/Versions/A/Lexicon
    0x7fff64b2b000 -     0x7fff64b32ff7  com.apple.LinguisticData (1.0 -
238.3) <49A54649-1021-3DBD-99B8-1B2EDFFA5378>
/System/Library/PrivateFrameworks/LinguisticData.framework/Versions/A/LinguisticData
    0x7fff65844000 -     0x7fff658adff7  com.apple.gpusw.MetalTools (1.0 -
1) <458F319A-2707-3C83-8351-BD9F02EC05BD>
/System/Library/PrivateFrameworks/MetalTools.framework/Versions/A/MetalTools
    0x7fff65a2d000 -     0x7fff65a46fff  com.apple.MobileKeyBag (2.0 - 1.0)
<32E63C7B-E133-33DE-A593-C3C10D64FCAA>
/System/Library/PrivateFrameworks/MobileKeyBag.framework/Versions/A/MobileKeyBag
    0x7fff65ad2000 -     0x7fff65afcffb
 com.apple.MultitouchSupport.framework (1404.4 - 1404.4)
<45374A2A-C0BC-3A70-8183-37295205CDFA>
/System/Library/PrivateFrameworks/MultitouchSupport.framework/Versions/A/MultitouchSupport
    0x7fff65d63000 -     0x7fff65d6efff  com.apple.NetAuth (6.2 - 6.2)
<B3795F63-C14A-33E1-9EE6-02A2E7661321>
/System/Library/PrivateFrameworks/NetAuth.framework/Versions/A/NetAuth
    0x7fff67604000 -     0x7fff67614ffb  com.apple.PerformanceAnalysis
(1.194 - 194) <2844933E-B71C-3BE9-9A84-27B29E111F13>
/System/Library/PrivateFrameworks/PerformanceAnalysis.framework/Versions/A/PerformanceAnalysis
    0x7fff693d3000 -     0x7fff693f1fff  com.apple.ProtocolBuffer (1 - 260)
<40704740-4A53-3010-A49B-08D1D69D1D5E>
/System/Library/PrivateFrameworks/ProtocolBuffer.framework/Versions/A/ProtocolBuffer
    0x7fff6af0a000 -     0x7fff6b01fff7  com.apple.Sharing (1050.22.2 -
1050.22.2) <4E3CCDF2-EA26-334F-8EBA-79BD28486C9D>
/System/Library/PrivateFrameworks/Sharing.framework/Versions/A/Sharing
    0x7fff6b04a000 -     0x7fff6b04bff7
 com.apple.performance.SignpostNotification (1.2.6 - 2.6)
<8F04800F-3570-3392-A24D-B229FF03F7F9>
/System/Library/PrivateFrameworks/SignpostNotification.framework/Versions/A/SignpostNotification
    0x7fff6bd93000 -     0x7fff6c02fff7  com.apple.SkyLight (1.600.0 -
312.103) <27F91170-846C-3E9E-9B8A-788F27C7DAF5>
/System/Library/PrivateFrameworks/SkyLight.framework/Versions/A/SkyLight
    0x7fff6c7f8000 -     0x7fff6c805fff  com.apple.SpeechRecognitionCore
(4.6.1 - 4.6.1) <87EE7AB5-6925-3D21-BE00-F155CB457699>
/System/Library/PrivateFrameworks/SpeechRecognitionCore.framework/Versions/A/SpeechRecognitionCore
    0x7fff6d3ab000 -     0x7fff6d434fc7  com.apple.Symbolication (9.3 -
64033) <C2C55C9A-C264-3044-A953-16457148190A>
/System/Library/PrivateFrameworks/Symbolication.framework/Versions/A/Symbolication
    0x7fff6d9a5000 -     0x7fff6d9adff7  com.apple.TCC (1.0 - 1)
<E1EB7272-FE6F-39AB-83CA-B2B5F2A88D9B>
/System/Library/PrivateFrameworks/TCC.framework/Versions/A/TCC
    0x7fff6dbba000 -     0x7fff6dc77ff7  com.apple.TextureIO (3.7 - 3.7)
<F8BAC954-405D-3CC3-AB7B-048C866EF980>
/System/Library/PrivateFrameworks/TextureIO.framework/Versions/A/TextureIO
    0x7fff6f5e0000 -     0x7fff6f5e2ffb  com.apple.loginsupport (1.0 - 1)
<D1232C1B-80EA-3DF8-9466-013695D0846E>
/System/Library/PrivateFrameworks/login.framework/Versions/A/Frameworks/loginsupport.framework/Versions/A/loginsupport
    0x7fff6f749000 -     0x7fff6f77cff7  libclosured.dylib (551.4)
<3FB6B209-51F4-38DA-B1D8-2EE29D5BDD83> /usr/lib/closure/libclosured.dylib
    0x7fff6f836000 -     0x7fff6f86fff7  libCRFSuite.dylib (41)
<FE5EDB68-2593-3C2E-BBAF-1C52D206F296> /usr/lib/libCRFSuite.dylib
    0x7fff6f870000 -     0x7fff6f87bfff  libChineseTokenizer.dylib (28)
<53633C9B-A3A8-36F7-A53C-432D802F4BB8> /usr/lib/libChineseTokenizer.dylib
    0x7fff6f90d000 -     0x7fff6f90eff3  libDiagnosticMessagesClient.dylib
(104) <9712E980-76EE-3A89-AEA6-DF4BAF5C0574>
/usr/lib/libDiagnosticMessagesClient.dylib
    0x7fff6f945000 -     0x7fff6fb0fff3  libFosl_dynamic.dylib (17.8)
<C58ED77A-4986-31C2-994C-34DDFB8106F0> /usr/lib/libFosl_dynamic.dylib
    0x7fff6fb47000 -     0x7fff6fb47fff  libOpenScriptingUtil.dylib (174)
<610F0242-7CE5-3C86-951B-B646562694AF> /usr/lib/libOpenScriptingUtil.dylib
    0x7fff6fc83000 -     0x7fff6fc84ffb  libSystem.B.dylib (1252.50.4)
<CD555F3B-FDDB-35E5-A2FB-FBBF3D62031A> /usr/lib/libSystem.B.dylib
    0x7fff6fd17000 -     0x7fff6fd17fff  libapple_crypto.dylib (109.50.14)
<48BA2E76-BF2F-3522-A54E-D7FB7EAF7A57> /usr/lib/libapple_crypto.dylib
    0x7fff6fd18000 -     0x7fff6fd2eff7  libapple_nghttp2.dylib (1.24)
<01402BC4-4822-3676-9C80-50D83F816424> /usr/lib/libapple_nghttp2.dylib
    0x7fff6fd2f000 -     0x7fff6fd59ff3  libarchive.2.dylib (54)
<8FC28DD8-E315-3C3E-95FE-D1D2CBE49888> /usr/lib/libarchive.2.dylib
    0x7fff6fd5a000 -     0x7fff6fe5dfe7  libate.dylib (1.13.1)
<DABEA1B1-806C-34C9-8AFF-DEB6AB2829E3> /usr/lib/libate.dylib
    0x7fff6fe61000 -     0x7fff6fe61ff3  libauto.dylib (187)
<A05C7900-F8C7-3E75-8D3F-909B40C19717> /usr/lib/libauto.dylib
    0x7fff6fe62000 -     0x7fff6ff1aff3  libboringssl.dylib (109.50.14)
<E6813F87-B5E4-3F7F-A725-E6A7F2BD02EC> /usr/lib/libboringssl.dylib
    0x7fff6ff1b000 -     0x7fff6ff2bff3  libbsm.0.dylib (39)
<6BC96A72-AFBE-34FD-91B1-748A530D8AE6> /usr/lib/libbsm.0.dylib
    0x7fff6ff2c000 -     0x7fff6ff39ffb  libbz2.1.0.dylib (38)
<0A5086BB-4724-3C14-979D-5AD4F26B5B45> /usr/lib/libbz2.1.0.dylib
    0x7fff6ff3a000 -     0x7fff6ff90fff  libc++.1.dylib (400.9)
<7D3DACCC-3804-393C-ABC1-1A580FD00CB6> /usr/lib/libc++.1.dylib
    0x7fff6ff91000 -     0x7fff6ffb5ff7  libc++abi.dylib (400.8.2)
<EF5E37D7-11D9-3530-BE45-B986612D13E2> /usr/lib/libc++abi.dylib
    0x7fff6ffb7000 -     0x7fff6ffc7fff  libcmph.dylib (6)
<A5509EE8-7E00-3224-8814-015B077A3CF5> /usr/lib/libcmph.dylib
    0x7fff6ffc8000 -     0x7fff6ffdffd3  libcompression.dylib (47.60.2)
<CC7ACB0A-49CD-3564-978F-49F843186F27> /usr/lib/libcompression.dylib
    0x7fff7028a000 -     0x7fff702a2ff7  libcoretls.dylib (155.50.1)
<D350052E-DC4D-3185-ADBA-BA48EDCEE955> /usr/lib/libcoretls.dylib
    0x7fff702a3000 -     0x7fff702a4ff3  libcoretls_cfhelpers.dylib
(155.50.1) <B297F5D8-F2FE-3566-A752-E9D998B9C039>
/usr/lib/libcoretls_cfhelpers.dylib
    0x7fff70775000 -     0x7fff707cbff3  libcups.2.dylib (462.2.4)
<908099FB-C70E-38FA-9573-88CB98FDDE29> /usr/lib/libcups.2.dylib
    0x7fff7090b000 -     0x7fff7090bfff  libenergytrace.dylib (16)
<A92AB8B8-B986-3CE6-980D-D55090FEF387> /usr/lib/libenergytrace.dylib
    0x7fff70942000 -     0x7fff70947ff3  libheimdal-asn1.dylib (520.50.6)
<E358445A-B84E-31B5-BCCD-7E1397519D96> /usr/lib/libheimdal-asn1.dylib
    0x7fff70973000 -     0x7fff70a64ff7  libiconv.2.dylib (51.50.1)
<2FEC9707-3FAF-3828-A50D-8605086D060F> /usr/lib/libiconv.2.dylib
    0x7fff70a65000 -     0x7fff70c8cffb  libicucore.A.dylib (59180.0.1)
<34EBADD6-4092-30EC-90E8-F75241E94D76> /usr/lib/libicucore.A.dylib
    0x7fff70cd9000 -     0x7fff70cdafff  liblangid.dylib (128)
<39C39393-0D05-301D-93B2-F224FC4949AA> /usr/lib/liblangid.dylib
    0x7fff70cdb000 -     0x7fff70cf4ffb  liblzma.5.dylib (10)
<3D419A50-961F-37D2-8A01-3DC7AB7B8D18> /usr/lib/liblzma.5.dylib
    0x7fff70cf5000 -     0x7fff70d0bff7  libmarisa.dylib (9)
<D6D2D55D-1D2E-3442-B152-B18803C0ABB4> /usr/lib/libmarisa.dylib
    0x7fff70dbc000 -     0x7fff70fe4ff7  libmecabra.dylib (779.7.6)
<F462F170-E872-3D09-B219-973D5E99C09F> /usr/lib/libmecabra.dylib
    0x7fff711bc000 -     0x7fff71337fff  libnetwork.dylib (1229.70.2)
<E185D902-AC7F-3044-87C0-AE2887C59CE7> /usr/lib/libnetwork.dylib
    0x7fff713be000 -     0x7fff717af3b7  libobjc.A.dylib (723)
<BEEA51E2-1C4D-3672-87F6-F267B4AC6221> /usr/lib/libobjc.A.dylib
    0x7fff717c2000 -     0x7fff717c6fff  libpam.2.dylib (22)
<7B4D2CE2-1438-387A-9802-5CEEFBF26F86> /usr/lib/libpam.2.dylib
    0x7fff717c9000 -     0x7fff717fdfff  libpcap.A.dylib (79.20.1)
<FA13918B-A247-3181-B256-9B852C7BA316> /usr/lib/libpcap.A.dylib
    0x7fff7187c000 -     0x7fff71898ffb  libresolv.9.dylib (65)
<E8F3415B-4472-3202-8901-41FD87981DB2> /usr/lib/libresolv.9.dylib
    0x7fff718e9000 -     0x7fff71a76ff7  libsqlite3.dylib (274.8.1)
<B70A2908-E0F7-364E-952A-1D2D19787B89> /usr/lib/libsqlite3.dylib
    0x7fff71c4a000 -     0x7fff71caaff3  libusrtcp.dylib (1229.70.2)
<1E065228-D0E3-3808-9405-894056C6BEC0> /usr/lib/libusrtcp.dylib
    0x7fff71cab000 -     0x7fff71caeffb  libutil.dylib (51.20.1)
<216D18E5-0BAF-3EAF-A38E-F6AC37CBABD9> /usr/lib/libutil.dylib
    0x7fff71caf000 -     0x7fff71cbcfff  libxar.1.dylib (400)
<0316128D-3B47-3052-995D-97B4FE5491DC> /usr/lib/libxar.1.dylib
    0x7fff71cc0000 -     0x7fff71da7fff  libxml2.2.dylib (31.11)
<C2B5C43F-9C0B-31E6-8EC0-939591EDAC49> /usr/lib/libxml2.2.dylib
    0x7fff71da8000 -     0x7fff71dd0fff  libxslt.1.dylib (15.12)
<4A5E011D-8B29-3135-A52B-9A9070ABD752> /usr/lib/libxslt.1.dylib
    0x7fff71dd1000 -     0x7fff71de3ffb  libz.1.dylib (70)
<48C67CFC-940D-3857-8DAD-857774605352> /usr/lib/libz.1.dylib
    0x7fff71e7f000 -     0x7fff71e83ff7  libcache.dylib (80)
<092479CB-1008-3A83-BECF-E115F24D13C1> /usr/lib/system/libcache.dylib
    0x7fff71e84000 -     0x7fff71e8eff3  libcommonCrypto.dylib (60118.50.1)
<029F5985-9B6E-3DCB-9B96-FD007678C6A7> /usr/lib/system/libcommonCrypto.dylib
    0x7fff71e8f000 -     0x7fff71e96fff  libcompiler_rt.dylib (62)
<968B8E3F-3681-3230-9D78-BB8732024F6E> /usr/lib/system/libcompiler_rt.dylib
    0x7fff71e97000 -     0x7fff71ea0ffb  libcopyfile.dylib (146.50.5)
<3885083D-50D8-3EEC-B481-B2E605180D7F> /usr/lib/system/libcopyfile.dylib
    0x7fff71ea1000 -     0x7fff71f26fff  libcorecrypto.dylib (562.70.1)
<5C26364F-2269-31EC-84AF-0FED2C902E38> /usr/lib/system/libcorecrypto.dylib
    0x7fff71fae000 -     0x7fff71fe7ff7  libdispatch.dylib (913.60.2)
<232C69BD-022E-3AB9-8807-79F9FA7CB5EC> /usr/lib/system/libdispatch.dylib
    0x7fff71fe8000 -     0x7fff72005ff7  libdyld.dylib (551.4)
<81BF3A82-5719-3B54-ABA9-76C82D932CAC> /usr/lib/system/libdyld.dylib
    0x7fff72006000 -     0x7fff72006ffb  libkeymgr.dylib (28)
<E34E283E-90FA-3C59-B48E-1277CDB9CDCE> /usr/lib/system/libkeymgr.dylib
    0x7fff72007000 -     0x7fff72013ff3  libkxld.dylib (4570.71.2)
<C3C31E1B-3E74-3828-8429-4D442E26D41C> /usr/lib/system/libkxld.dylib
    0x7fff72014000 -     0x7fff72014ff7  liblaunch.dylib (1205.70.9)
<B184B521-FF24-3142-AFAF-23D170CF918C> /usr/lib/system/liblaunch.dylib
    0x7fff72015000 -     0x7fff72019ffb  libmacho.dylib (906)
<1902A611-081A-3452-B11E-EBD1B166E831> /usr/lib/system/libmacho.dylib
    0x7fff7201a000 -     0x7fff7201cff3  libquarantine.dylib (86)
<26C0BA22-8F93-3A07-9A4E-C8D53D2CE42E> /usr/lib/system/libquarantine.dylib
    0x7fff7201d000 -     0x7fff7201eff3  libremovefile.dylib (45)
<711E18B2-5BBE-3211-A916-56740C27D17A> /usr/lib/system/libremovefile.dylib
    0x7fff7201f000 -     0x7fff72036fff  libsystem_asl.dylib (356.70.1)
<39E46A6F-B228-3E78-B83E-1779F9707A39> /usr/lib/system/libsystem_asl.dylib
    0x7fff72037000 -     0x7fff72037fff  libsystem_blocks.dylib (67)
<17303FDF-0D2D-3963-B05E-B4DF63052D47>
/usr/lib/system/libsystem_blocks.dylib
    0x7fff72038000 -     0x7fff720c1ff7  libsystem_c.dylib (1244.50.9)
<1187BFE8-4576-3247-8177-481554E1F9E7> /usr/lib/system/libsystem_c.dylib
    0x7fff720c2000 -     0x7fff720c5ffb  libsystem_configuration.dylib
(963.50.8) <DF6B5287-203E-30CB-9947-78DF446C72B8>
/usr/lib/system/libsystem_configuration.dylib
    0x7fff720c6000 -     0x7fff720c9ffb  libsystem_coreservices.dylib (51)
<486000D3-D8CB-3BE7-8EE5-8BF380DE6DF7>
/usr/lib/system/libsystem_coreservices.dylib
    0x7fff720ca000 -     0x7fff720cbfff  libsystem_darwin.dylib (1244.50.9)
<09C21A4A-9EE0-388B-A9D9-DFF8F6758791>
/usr/lib/system/libsystem_darwin.dylib
    0x7fff720cc000 -     0x7fff720d2ff7  libsystem_dnssd.dylib (878.70.2)
<3290768B-54DE-3AB6-B155-AC0950AC5564> /usr/lib/system/libsystem_dnssd.dylib
    0x7fff720d3000 -     0x7fff7211cff7  libsystem_info.dylib (517.30.1)
<AB634A98-B8AA-3804-8436-38261FC8EC4D> /usr/lib/system/libsystem_info.dylib
    0x7fff7211d000 -     0x7fff72143ff7  libsystem_kernel.dylib (4570.71.2)
<F22B8D73-69D8-36D7-BF66-7F9AC70C08C2>
/usr/lib/system/libsystem_kernel.dylib
    0x7fff72144000 -     0x7fff7218ffcb  libsystem_m.dylib (3147.50.1)
<8CFB51C9-B422-3379-8552-064C63943A23> /usr/lib/system/libsystem_m.dylib
    0x7fff72190000 -     0x7fff721affff  libsystem_malloc.dylib (140.50.6)
<7FD43735-9DDD-300E-8C4A-F909A74BDF49>
/usr/lib/system/libsystem_malloc.dylib
    0x7fff721b0000 -     0x7fff722e0ff7  libsystem_network.dylib
(1229.70.2) <5E86B2DE-9E15-3354-8714-4094ED5F698D>
/usr/lib/system/libsystem_network.dylib
    0x7fff722e1000 -     0x7fff722ebffb  libsystem_networkextension.dylib
(767.70.1) <D23EAFC1-E8BD-34D5-969C-6E45A1C3B4E4>
/usr/lib/system/libsystem_networkextension.dylib
    0x7fff722ec000 -     0x7fff722f5ff3  libsystem_notify.dylib (172)
<08012EC0-2CD2-34BE-BF93-E7F56491299A>
/usr/lib/system/libsystem_notify.dylib
    0x7fff722f6000 -     0x7fff722fdff7  libsystem_platform.dylib
(161.50.1) <6355EE2D-5456-3CA8-A227-B96E8F1E2AF8>
/usr/lib/system/libsystem_platform.dylib
    0x7fff722fe000 -     0x7fff72309fff  libsystem_pthread.dylib (301.50.1)
<0E51CCBA-91F2-34E1-BF2A-FEEFD3D321E4>
/usr/lib/system/libsystem_pthread.dylib
    0x7fff7230a000 -     0x7fff7230dfff  libsystem_sandbox.dylib (765.70.1)
<553DFCC6-9D31-3B9C-AB7C-30F6F265786D>
/usr/lib/system/libsystem_sandbox.dylib
    0x7fff7230e000 -     0x7fff7230fff3  libsystem_secinit.dylib (30)
<DE8D14E8-A276-3FF8-AE13-77F7040F33C1>
/usr/lib/system/libsystem_secinit.dylib
    0x7fff72310000 -     0x7fff72317ff7  libsystem_symptoms.dylib
(820.60.2) <585BDFA2-D54D-39D0-8046-44E824DABD43>
/usr/lib/system/libsystem_symptoms.dylib
    0x7fff72318000 -     0x7fff7232bfff  libsystem_trace.dylib (829.70.1)
<3A6CB706-8CA6-3616-8AFC-14AAD7FAF187> /usr/lib/system/libsystem_trace.dylib
    0x7fff7232d000 -     0x7fff72332ff7  libunwind.dylib (35.3)
<BEF3FB49-5604-3B5F-82B5-332B80023AC3> /usr/lib/system/libunwind.dylib
    0x7fff72333000 -     0x7fff72360ff7  libxpc.dylib (1205.70.9)
<0BC7AD67-671D-31D4-8B88-C317B8379598> /usr/lib/system/libxpc.dylib

External Modification Summary:
  Calls made by other processes targeting this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by all processes on this machine:
    task_for_pid: 39839
    thread_create: 0
    thread_set_state: 0

VM Region Summary:
ReadOnly portion of Libraries: Total=321.0M resident=0K(0%)
swapped_out_or_unallocated=321.0M(100%)
Writable regions: Total=192.6M written=0K(0%) resident=0K(0%)
swapped_out=0K(0%) unallocated=192.6M(100%)

                                VIRTUAL   REGION
REGION TYPE                        SIZE    COUNT (non-coalesced)
===========                     =======  =======
Activity Tracing                   256K        2
CoreServices                      17.5M        2
Kernel Alloc Once                    8K        2
MALLOC                           164.4M       30
MALLOC guard page                   32K        9
STACK GUARD                       56.0M        6
Stack                             10.0M        6
VM_ALLOCATE                         36K        3
__DATA                            17.9M      214
__FONT_DATA                          4K        2
__LINKEDIT                       197.6M        5
__TEXT                           123.3M      218
__UNICODE                          560K        2
mapped file                       25.8M        3
shared memory                     4704K        9
===========                     =======  =======
TOTAL                            618.0M      498

Model: MacPro5,1, BootROM MP51.0085.B00, 12 processors, 6-Core Intel Xeon,
3.46 GHz, 96 GB, SMC 1.39f5
Graphics: NVIDIA GeForce GTX 1060 6GB, NVIDIA GeForce GTX 1060 6GB, PCIe
Memory Module: DIMM 1, 16 GB, DDR3 ECC, 1333 MHz, 0x80CE,
0x4D33393342324737304442302D594B302020
Memory Module: DIMM 2, 16 GB, DDR3 ECC, 1333 MHz, 0x80CE,
0x4D33393342324737304442302D594B302020
Memory Module: DIMM 3, 16 GB, DDR3 ECC, 1333 MHz, 0x80AD,
0x484D5434324752374D465234412D50422020
Memory Module: DIMM 5, 16 GB, DDR3 ECC, 1333 MHz, 0x80CE,
0x4D33393342324737304442302D594B302020
Memory Module: DIMM 6, 16 GB, DDR3 ECC, 1333 MHz, 0x80CE,
0x4D33393342324737304442302D594B302020
Memory Module: DIMM 7, 16 GB, DDR3 ECC, 1333 MHz, 0x80AD,
0x484D5434324752374D465234412D50422020
Bluetooth: Version 6.0.7f10, 3 services, 27 devices, 1 incoming serial ports
Network Service: Ethernet 1, Ethernet, en0
Network Service: Ethernet 2, Ethernet, en1
PCI Card: NVIDIA GeForce GTX 1060 6GB, Display Controller, Slot-1
PCI Card: pci10de,10f1, Audio Device, Slot-1
PCI Card: PXS4, USB eXtensible Host Controller, Slot-4
PCI Card: pci1b4b,9182, AHCI Controller, Slot-3@5,0,0
PCI Card: pci1b4b,9182, AHCI Controller, Slot-3@6,0,0
Serial ATA Device: HL-DT-ST DVD-RW GH41N
Serial ATA Device: WDC WD30EFRX-68EUZN0, 3 TB
Serial ATA Device: WDC WD20EZRX-00D8PB0, 2 TB
Serial ATA Device: WDC WD30EFRX-68EUZN0, 3 TB
Serial ATA Device: WDC WD30EFRX-68EUZN0, 3 TB
Serial ATA Device: WDC WD30EFRX-68EUZN0, 3 TB
Serial ATA Device: Samsung SSD 850 EVO 500GB, 500.11 GB
Serial ATA Device: Crucial_CT240M500SSD1, 240.06 GB
USB Device: USB Bus
USB Device: USB Bus
USB Device: BRCM2046 Hub
USB Device: Bluetooth USB Host Controller
USB Device: USB Bus
USB Device: USB Bus
USB Device: Intuos5 touch S
USB Device: USB Bus
USB Device: USB Bus
USB Device: USB 2.0 Bus
USB Device: USB2.0 Hub
USB Device: USB 2.0 Bus
USB Device: Pro9000
USB Device: USB 3.0 Bus
USB Device: USB3.0 Hub
USB Device: USB3.0 Hub
USB Device: USB Storage
USB Device: BUP BK
USB Device: M3 Portable
USB Device: USB3.1 Device
USB Device: USB2.0 Hub
USB Device: USB2.0 Hub
USB Device: Hub
USB Device: EIZO USB HID Monitor
FireWire Device: built-in_hub, Up to 800 Mb/sec
Thunderbolt Bus:

**Comment by crwood** [comment]
Hmm... That's perplexing. If I had to hazard a guess, I'd say that one of the libraries in the bundle was compiled with a processor-specific optimization that's absent from your Xeon processor (but, judging from the crashlog output alone, I have no idea which...).

In the meantime, could you paste the output of `sysctl -a | grep machdep.cpu `?

Also, if you're willing to trust/test it, there's a development build of Gridsync available [here](https://buildbot.gridsync.io/artifacts/Gridsync-Mac.dmg) that was compiled on much older hardware (and which contains some upcoming/experimental features) -- I'd be curious to know whether that build would run on your machine..

**Comment by eleaner** [comment]
The development version is working
two questions
1. how can I share the same sync folders between multiple computers? or
view it in the web interface of another client?
2. I see app displays the available space. That's the total space in the
grid.
How can I get available storage space?


see the requested output below

machdep.cpu.max_basic: 11
machdep.cpu.max_ext: 2147483656
machdep.cpu.vendor: GenuineIntel
machdep.cpu.brand_string: Intel(R) Xeon(R) CPU           X5690  @ 3.47GHz
machdep.cpu.family: 6
machdep.cpu.model: 44
machdep.cpu.extmodel: 2
machdep.cpu.extfamily: 0
machdep.cpu.stepping: 2
machdep.cpu.feature_bits: 188838921972218879
machdep.cpu.extfeature_bits: 5034215680
machdep.cpu.signature: 132802
machdep.cpu.brand: 0
machdep.cpu.features: FPU VME DE PSE TSC MSR PAE MCE CX8 APIC SEP MTRR PGE
MCA CMOV PAT PSE36 CLFSH DS ACPI MMX FXSR SSE SSE2 SS HTT TM PBE SSE3
PCLMULQDQ DTES64 MON DSCPL VMX SMX EST TM2 SSSE3 CX16 TPR PDCM SSE4.1
SSE4.2 POPCNT AES PCID
machdep.cpu.extfeatures: SYSCALL XD 1GBPAGE EM64T LAHF RDTSCP TSCI
machdep.cpu.logical_per_package: 32
machdep.cpu.cores_per_package: 16
machdep.cpu.microcode_version: 15
machdep.cpu.processor_flag: 0
machdep.cpu.mwait.linesize_min: 64
machdep.cpu.mwait.linesize_max: 64
machdep.cpu.mwait.extensions: 3
machdep.cpu.mwait.sub_Cstates: 4384
machdep.cpu.thermal.sensor: 1
machdep.cpu.thermal.dynamic_acceleration: 1
machdep.cpu.thermal.invariant_APIC_timer: 1
machdep.cpu.thermal.thresholds: 2
machdep.cpu.thermal.ACNT_MCNT: 1
machdep.cpu.thermal.core_power_limits: 0
machdep.cpu.thermal.fine_grain_clock_mod: 0
machdep.cpu.thermal.package_thermal_intr: 0
machdep.cpu.thermal.hardware_feedback: 0
machdep.cpu.thermal.energy_policy: 0
machdep.cpu.arch_perf.version: 3
machdep.cpu.arch_perf.number: 4
machdep.cpu.arch_perf.width: 48
machdep.cpu.arch_perf.events_number: 7
machdep.cpu.arch_perf.events: 4
machdep.cpu.arch_perf.fixed_number: 3
machdep.cpu.arch_perf.fixed_width: 48
machdep.cpu.cache.linesize: 64
machdep.cpu.cache.L2_associativity: 8
machdep.cpu.cache.size: 256
machdep.cpu.tlb.inst.small: 64
machdep.cpu.tlb.inst.large: 7
machdep.cpu.tlb.data.small: 64
machdep.cpu.tlb.data.large: 32
machdep.cpu.tlb.shared: 512
machdep.cpu.address_bits.physical: 40
machdep.cpu.address_bits.virtual: 48
machdep.cpu.core_count: 6
machdep.cpu.thread_count: 12
machdep.cpu.tsc_ccc.numerator: 0
machdep.cpu.tsc_ccc.denominator: 0

**Comment by eleaner** [comment]
Please forget my sharing folder question. Stupid of me to ask
But I have another bunch of questions (if you don't mind)
I decided to go with a manual configuration. There was an option to provide introducer but not the helper. How can I add it? 
In fact, how can I access the web interface of running? I'd like to see the stats.  Should it be available under standard port 3456? I had running another version of tahoe when I started gridsync so I am not so sure if it simply failed to initialise.
Actually there a way to see the standard tahoe config files and URIs of the synced folders?

thanks
Marcin

**Comment by crwood** [comment]
I'm glad to hear the development build worked on your machine! Thank you so much for testing it! :)

I've just opened up [a separate issue](https://github.com/gridsync/gridsync/issues/110) for distributing binaries that will run on older machines. I'll be sure to have such builds in place for the next (0.4) release.

To address your other questions:

You can share a folder by right-clicking on it in the main window and selecting the "Share" option. This will create a temporary/one-time-use "invite code" (using the [magic-wormhole](https://github.com/warner/magic-wormhole) library) which, when entered on a second computer, will establish a secure connection between the two machines through which the necessary credentials will be transferred (in this case, the introducer/storage fURLs and other connection-related information needed to join the grid along with the magic-folder capabilities needed to join the folder(s)). Currently, due to the risk of uncordinated write errors with Tahoe-LAFS's mutables, you can only share folders that you've added/created (i.e., you can't reshare a folder that somebody else shared with you).

I'm not sure if I follow your second question correctly (and please let me know if I've misinterpreted you), but the amount of available storage space depends on the grid to which you are connected (and can fluctuate, e.g., depending on whether other users are uploading files, whether additional storage nodes join/leave, etc.) and so to get more storage space you can, e.g., join a grid that offers it, create/configure a storage grid of your own, or buy storage from a commercial provider (like [Least Authority](https://leastauthority.com/) -- who funds Gridsync and for whom I work).

I did not add an option to specify a helper fURL since I've yet come across anybody who actually uses one... Do you use a helper? Would it be useful to you if Gridsync supported such an option via the manual configuration UI? In any case, you can add an introducer by hand, if you'd like, by editing the `tahoe.cfg` file for the grid in question (which, on macOS, resides at `~/Library/Application Support/Gridsync/GRID_NAME_HERE/tahoe.cfg`) and then restarting Gridsync (so that the updated settings take effect).

I originally decided not to provide a means of accessing the Tahoe-LAFS web UI from within Gridsync since a) there is no way to add/join/configure magic-folders via the web UI (which Gridsync uses almost exclusively) and b) I don't want to introduce the additional risk of exposing the user's capabilities to potentially-malicious browsers/webpages (though, in retrospect, I agree that it may still be useful to make such functionality available after, say, an appropriate warning). That said, if you'd like to view/use the Tahoe-LAFS web UI, for now, you'll have to open in manually via the tahoe CLI (e.g., with the command `Gridsync.app/Contents/MacOS/Tahoe-LAFS/tahoe -d ~/Library/Application Support/Gridsync/GRID_NAME_HERE webopen`) or by editing the `tahoe.cfg` file mentioned above to listen on a specific port so that you can navigate to it directly with your browser (as any tahoe clients configured by Gridsync will listen on a random available port so as to not interfere with existing running processes). 

All other standard Tahoe-LAFS configuration/data files (including the databases containing your magic-folder caps, etc.) can be found under the Gridsync configuration directory at `~/Library/Application Support/Gridsync`

Please don't hesitate to ask should you have any other questions!

**Comment by eleaner** [comment]
Thank you for all the answers.
I am unable to test it now. That's sort-of weekend project.
and yes I am using a helper. In my case, it speeds up uploads massively :)

**Comment by crwood** [comment]
Closing this; with Gridsync 0.4, a ["Legacy" macOS build](https://github.com/gridsync/gridsync/releases/tag/v0.4.0) has been provided that should run on hardware as old as 2009 on macOS 10.9 and above.

Please feel free to open a new issue if this build isn't working on your machine -- or if you run into any other problems. Thanks again!
